### PR TITLE
Import WebExtensions API JSONSchema data from Gecko 154.0b4

### LINKED ISSUE #6126: Import WebExtensions API JSONSchema data from Gecko 154
This is about importing the schemas for Fx154.

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-linter/issues/6126

### CODE REVIEW

File: src/schema/validator.js [hunk 1]
```diff
@@ -489,6 +489,9 @@ export class SchemaValidator {
       'imageDataOrStrictRelativeUrl',
       imageDataOrStrictRelativeUrl
     );
+
+    // TODO: maybe implement this validator
+    validator.addFormat('contentSecurityPolicySandbox', () => true);
```
Comment (Rob--W) [inline src/schema/validator.js]: We must verify that if set, it contains the `sandbox` directive without `allow-same-origin`. Extensions not adhering to these rules will not load in Firefox (or Chrome).

File: tests/unit/schema/test.formats.js [hunk 2]
```diff
@@ -312,4 +313,27 @@ describe('formats', () => {
...
   });
+
+  describe('contentSecurityPolicySandbox', () => {
+    it.each([
+      'sandbox',
+      'sandbox allow-scripts',
+      'SANDBOX ALLOW-SCRIPTS',
+      "sandbox allow-scripts; script-src 'self'",
+      "script-src 'self'; sandbox allow-scripts",
+      '  sandbox allow-scripts ;',
```
Comment (Rob--W) [inline tests/unit/schema/test.formats.js:324]:
```suggestion
      '  sandbox allow-scripts ;',
      'sandbox not-allow-same-origin',
      // First directive applied, duplicate ignored.
      'sandbox; sandbox allow-same-origin',
```

File: tests/unit/schema/test.formats.js [hunk 3]
```diff
@@ -312,4 +313,27 @@ describe('formats', () => {
...
+    ])('accepts %s', (value) => {
+      expect(contentSecurityPolicySandbox(value)).toEqual(true);
+    });
+
+    it.each([
+      '',
+      "script-src 'self'",
+      'sandbox allow-scripts allow-same-origin',
+      'sandbox allow-same-origin',
+      'sandbox ALLOW-SAME-ORIGIN',
```
Comment (Rob--W) [inline tests/unit/schema/test.formats.js:337]:
```suggestion
      'sandbox ALLOW-SAME-ORIGIN',
      // First applied, duplicate ignored.
      'sandbox allow-same-origin; sandbox',
```

**Review by Rob--W — APPROVED** [review]
Added some extra tests to cover some less trivial edge cases.
